terminologies/technique/multiContactLaminarElectrodeRecording.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/multiContactLaminarElectrodeRecording",

```

Comment:
do we want to have this consistent with the other terms? meaning "multi-contact laminar electrode electrophysiology"?